src/services/AttestationWorkflow.ts

```diff
@@ -137,22 +137,22 @@ class AttestationWorkflow {
       throw new Error('No selected Identity')
     }
 
-    const requestForAttestation = await RequestForAttestation.fromClaimAndIdentity(
+    const requestForAttestation = RequestForAttestation.fromClaimAndIdentity(
```

Comment:
Is the function `submitClaimsForCTypes` now still async?

Reply:
Yes, because we are fetching going deeper till we get to `MessageRepository.send` where we are waiting for a promise to resolve.

Reply:
But only because you return a function, which itself returns a promise, doesn't mean you have to use `async`.
You have to use async, when you either want to use `await` (which we do not use here anymore), or if you return a promise (which is the case here) and you might throw an error in this function.

Reply:
The error case is relevant, because someone using this function might use a `.catch()` function on the returned promise, instead of `try...catch`.